media-api/app/controllers/SuggestionController.scala

```diff
@@ -0,0 +1,108 @@
+package controllers
+
+import play.api.libs.concurrent.Execution.Implicits._
+import play.api.libs.json.{Reads, Json, Writes}
+import play.api.mvc.{AnyContent, Request, Controller}
+
+import com.gu.mediaservice.lib.argo.ArgoHelpers
+import com.gu.mediaservice.lib.auth
+import com.gu.mediaservice.lib.auth.{PermissionStore, KeyStore}
+
+import lib.Config
+import lib.elasticsearch.{AggregateSearchResults, ElasticSearch}
+import lib.querysyntax.Parser
+
+import scala.util.Try
+
+object SuggestionController extends Controller with ArgoHelpers {
+
+  val keyStore = new KeyStore(Config.keyStoreBucket, Config.awsCredentials)
```

Comment:
Seems odd to have a second keystore. Also it's never started in the Global file. Can we reuse the same keystore (and permission store) for both?